test/module/irohad/consensus/yac/yac_gate_test.cpp

```diff
@@ -88,6 +88,11 @@ class YacGateTest : public ::testing::Test {
                                          hash_provider,
                                          block_creator,
                                          block_cache);
+
+    auto peer = makePeer("127.0.0.1", shared_model::crypto::PublicKey("111"));
```

Comment:
Should not we use here a public key of proper length?

Reply:
Probably not, because there is no particular crypto algorithm used in the test, so there is no proper length in this context.